discourse-doctor

```diff
@@ -0,0 +1,374 @@
+#!/usr/bin/env bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $DIR
+
+LOG_FILE="/tmp/discourse-debug.txt"
+rm -f $LOG_FILE
+echo DISCOURSE DOCTOR `date` | tee -a $LOG_FILE
```

Comment:
I'm not bash script expert but I think it's better to write:

```
echo "DISCOURSE DOCTOR $(date)" | tee -a $LOG_FILE
```

Reply:
Well, my way was better 25 years ago, but it looks like you're right today. :-)

https://stackoverflow.com/questions/9405478/command-substitution-backticks-or-dollar-sign-paren-enclosed